config/testgrids/kubernetes/sig-windows/config.yaml

```diff
@@ -10,6 +11,7 @@ dashboard_groups:
     - sig-windows-networking
 
 dashboards:
+- name: sig-windows-signal
```

Comment:
I called it `signal` so we could add non informing jobs to it later if we wanted.  Not tied to the name